packages/studio-plugin/tests/__fixtures__/LayoutFile/BasicLayout.tsx

```diff
@@ -1,6 +1,5 @@
 import "./index.css";
 import ComplexBanner from "../ComponentFile/ComplexBanner";
-import "@yext/search-ui-react/index.css";
```

Comment:
can we try testing with `@yext/search-ui-react/bundle.css` if you need it to resolve to a real path?

Reply:
Discussed offline, will address this in another PR!